Composer/packages/client/src/pages/botProject/adapters/AdapterSection.tsx

```diff
@@ -72,7 +72,7 @@ const AdapterSection = ({ projectId, scrollToSectionId }: Props) => {
               a2: ({ children }) => (
                 <Link
                   key="package-adapter-settings-page-learn-more"
-                  href={'https://aka.ms/composer-packagmanageraddconnection-learnmore'}
+                  href={'https://docs.microsoft.com/en-us/composer/how-to-manage-packages?tabs=dotnet'}
```

Comment:
Sorry, I just realized the link is correct but we should link the Connect to package feed section.
https://docs.microsoft.com/en-us/composer/how-to-manage-packages?tabs=dotnet#connect-to-package-feeds

Reply:
Done. Thanks!